nodewatcher/modules/analysis/rogue_nodes/tests.py

```diff
@@ -0,0 +1,28 @@
+import json
+import io
+import os
+
+from django import test as django_test
+
+from nodewatcher.modules.analysis.rogue_nodes.tasks import rogue_node_detection_algorithm
+
+
+class RogueDetectionTest(django_test.TestCase):
+
+    def test_rogue_detection_algorithm(self):
+        print('testing rogue_node_detection_algorithm')
```

Comment:
So you define a custom test suite:

* https://stackoverflow.com/questions/9261474/custom-test-suite-for-django-app
* http://stackoverflow.com/a/20870875/252025
* http://stackoverflow.com/a/23508426/252025